modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/index.ts

```diff
@@ -130,9 +130,12 @@ export interface IQuickActions extends IBaseActions {
 	onClick: Function;
 }
 
-type TSorting = {
+export type TSorting = {
+	active?: boolean;
+	default?: boolean;
```

Comment:
I'm glad that you added `default`, although it doesn't seem to be used anywhere. It will be important down the line, when we have custom views that need to reset to default state.